xbmc/utils/ContentUtils.h

```diff
@@ -10,9 +10,11 @@
 
 #include "media/MediaType.h"
 
+#include <memory>
 #include <string>
 
 class CFileItem;
+using CFileItemPtr = std::shared_ptr<CFileItem>;
```

Comment:
Not sure this is a good idea as this type definition is already in FileItem.h. If the type behind this def gets changed here or at the other place, this can lead to problems (no longer compiling as the best case, one must not forget to change all places where this type is defined, ...). If you need CFileItemPtr like defined in FileItem.h, which I guess is your intention, you should include FileItem.h here.

I personally avoid using `CFileItemPtr`and directly use the full type `std::shared_ptr<CFileItem>`. Is not  much more to type, but avoids the problem i described above ... and you can forward declared CFileItem, no need to include the heavyweight FileItem.h.